### PR TITLE
update zaks personal project ideact.it to README

### PR DESCRIPTION
**Pull request by zakwatts**
Add Ideact.it to the personal project section.

### CODE REVIEW

**Comment by zakwatts** [comment]
small change made to personal project name and description below mine to make it clearer where to add a project

**Comment by zakwatts** [comment]
Screenshot 2023-04-11 at 23 13 20

I found doing "gem CONTRIBUTING.md" create a load of files. Is that normal?

**Comment by zakwatts** [comment]
I would add this as a task for you but I don't think you sent me a request to help manage the repo. But you should update how frequent office hours are on it and also add a link to where you think is the best place to direct people for that.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,21 +2,27 @@
 
 Contributions are welcome, and they are greatly appreciated!
 
+To contribute make sure to fork this repository, push your changes to that forked repository, then submit a pull request to this repository for your work to be merged.
+
 Every little bit helps, and credit will always be given.
-So feel free to create a [new merge/pull request][merge-link]!
 
-And make sure to follow the [guidelines](#merge-pull-request-guidelines).
+If you get stuck or you are unsure about something pop into the EA Software Engineers office hours for help.
```
Comment (NicoleJaneway) [inline CONTRIBUTING.md]: Can you capitalize Office Hours, add this link (https://docs.google.com/presentation/d/1T11dj1I3c23ZMLGWL-EYZVVJkgOv9coNR1355MWojOA/edit#slide=id.g219723c9878_0_39), and add that Office Hours are the second Monday of every month?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,21 +2,27 @@
 
 Contributions are welcome, and they are greatly appreciated!
 
+To contribute make sure to fork this repository, push your changes to that forked repository, then submit a pull request to this repository for your work to be merged.
```
Comment (NicoleJaneway) [inline CONTRIBUTING.md:5]: Good idea 🙂

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,21 +2,27 @@
...
+
+Please make sure to follow the [guidelines](#merge-pull-request-guidelines).
 
 ## Merge/Pull Request Guidelines
 
 Before you submit a pull request, check that it meets these guidelines:
+
 1. Make sure to have atomic commits and contextual commit messages!
-  [Check out this awesome blog post by Chris Beams for more information.][chris-beams]
+   [Check out this awesome blog post by Chris Beams for more information.][chris-beams]
```
Comment (NicoleJaneway) [inline CONTRIBUTING.md:18]: Whitespace error?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# EA Software Engineers Awesome List [![Awesome][awesome-badge]][awesome-link]
+# EA Software Engineers Awesome List [Awesome][awesome-link]
```
Comment (NicoleJaneway) [inline README.md]: Can we put this back?

File: README.md [hunk 5]
```diff
@@ -21,7 +21,8 @@ An unofficial list of personal and open source projects that are relevant to EA
 
 Created by EA Software Engineers
 
-- [Item](https://github.com/Pawamoy/cookiecutter-awesome) – short description goes here
+- [Ideact.it ](https://app.ideact.it/)– A tool to evaluate different cause areas and ideas by assigning significance values to different variables. The tool tries to combine the ITN framework with personal factors and give a clear visulisation of this. (Zak)
```
Comment (NicoleJaneway) [inline README.md:24]: Yay!

File: README.md [hunk 6]
```diff
@@ -21,7 +21,8 @@ An unofficial list of personal and open source projects that are relevant to EA
...
+- [Your Project Name]() – A short description about your project
```
Comment (NicoleJaneway) [inline README.md:25]: Good idea!!

File: README.md [hunk 7]
```diff
@@ -49,15 +50,17 @@ Recommended by EA Software Engineers
...
 
-Contributions welcome! Read the [contribution guidelines](CONTRIBUTING.md) first.
+Contributions are welcome!
+
+Find out how to contribute and read the [contribution guidelines](CONTRIBUTING.md) here.
 
 ## Credits
 
-This project was initially created with [Cookiecutter][cookiecutter] and the custom [cookiecutter-awesome][cookiecutter-awesome] :cookie:
+This project was initially created with [Cookiecutter][cookiecutter] and the custom [cookiecutter-awesome][cookiecutter-awesome] 🍪
```
Comment (NicoleJaneway) [inline README.md]: please revert

File: README.md [hunk 8]
```diff
@@ -49,15 +50,17 @@ Recommended by EA Software Engineers
...
 
 ## License
 
-[![CC0][cc0-badge]][cc0-link]
+[CC0][cc0-link]
```
Comment (NicoleJaneway) [inline README.md]: please revert

**Review by NicoleJaneway — CHANGES_REQUESTED** [review]

**Review by NicoleJaneway — APPROVED** [review]

**Pull request merged**